src/ui/components/TestSuite/views/TestRecording/Panel.tsx

```diff
@@ -40,6 +47,27 @@ export default function Panel() {
     committedValuesRef.current.testEvent = testEvent;
   });
 
+  // We only want to cache the test event details when the focus window has been updated
+  // to match the range of the test recording.  Experimentation shows there can be some renders
+  // where the focus range and test range are mismatched, so try to avoid caching in those cases.
+  const enableCache =
+    focusWindow &&
+    testRecording.timeStampedPointRange?.begin &&
+    isExecutionPointsWithinRange(
+      focusWindow.begin.point,
+      testRecording.timeStampedPointRange.begin.point,
+      testRecording.timeStampedPointRange.begin.point
+    );
+
+  useImperativeIntervalCacheValues(
```

Comment:
I think this will trigger rerendering this component when the details come in, but that is unnecessary (I think). You could just call `testEventDetailsIntervalCache.readAsync(...)` (without awaiting, obviously).

Reply:
Eh, technically not necessary I guess, but calling `.readAsync()` directly while rendering is weird :)  (then again so is Suspense? 🤷‍♂️  )